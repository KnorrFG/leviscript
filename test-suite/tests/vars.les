let first_word = "ababua"
let name = "Levi"
let mystr = "${name}'s first word was: $first_word"
x{echo output: $mystr}
