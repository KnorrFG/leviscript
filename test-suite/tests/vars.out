output: Levi's first word was: ababua
